document/Definitions.mdk

```diff
@@ -45,49 +48,49 @@ rule metadata
   ~ Example: Category (for example, "Style" or "Security"), documentation URL 
 
 log file
-  ~ output file produced by a static analysis tool, which enumerates the issues detected by the tool
+  ~ output file produced by a static analysis tool, which enumerates the results produced by the tool
 
 run
   ~ 1. invocation of a specified set of static analysis tools on a specified version of a specified set of analysis targets,
        with a specified set of runtime parameters
-  ~ 2. set of issues produced by such an invocation.
+  ~ 2. set of results produced by such an invocation.
 
 triage
-  ~ process of deciding whether an issue reported by a static analysis tool should be corrected
+  ~ process of deciding whether a result reported by a static analysis tool indicates a problem that should be corrected
 
 (end) user
-  ~ person who uses the information in a log file to diagnose, triage, or resolve issues detected by a static analysis tool
+  ~ person who uses the information in a log file to diagnose, triage, or resolve results detected by a static analysis tool
```

Comment:
you didn't introduce this language but I think 'investigate' is better than 'diagnose' here